src/test/suite/fixtures/recorded/languages/java/changeCallee.yml

```diff
@@ -0,0 +1,23 @@
+languageId: java
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: obj.addNumbers(num1, num2);
+  selections:
+    - anchor: {line: 0, character: 24}
+      active: {line: 0, character: 24}
+  marks: {}
+finalState:
+  documentContents: obj.(num1, num2);
```

Comment:
This is wrong imo.  Should select `obj.addNumbers`.  cc/ @AndreasArvidsson tho in case his expectations for Java would be different

Reply:
I'm unsure. I don't think I really understand what this scope type is supposed to do.

Reply:
I've removed these bad tests.